WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -423,6 +431,10 @@ private void checkAndSend() {
             return;
         }
 
+        if (!NetworkUtils.checkConnection(getActivity())) {
+            return;
```

Comment:
I'd suggest we let the user know that the action was not performed due to lack of connectivity or something. Wdyt?